Evaluate XiHe and WenHai

ZakariaAissaabdi opened the pull request.

qgau commented on oceanbench/core/references/glorys.py:
```diff
@@ -4,22 +4,29 @@
 from oceanbench.core.dataset_utils import Dimension
 
+os.environ["COPERNICUSMARINE_USERNAME"] = "zaissa@mercator-ocean.fr"
+os.environ["COPERNICUSMARINE_PASSWORD"] = "Banana-!31100"
```
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

Please review change BEFORE opening a MR.
Change your password.
Never commit a password.

ZakariaAissaabdi commented:
I took the remarks into consideration and modified my code, I also rebased on main resolving conflicts locally, maybe the branch is ready for merging now
